src/tclmisc.c

```diff
@@ -303,6 +303,29 @@ static int tcl_utimer STDVAR
   Tcl_AppendResult(irp, x, NULL);
   return TCL_OK;
 }
+static int tcl_timerexists STDVAR
+{
+  BADARGS(2, 2, " timerName");
+
+  if (find_timer(timer, argv[1])) {
```

Comment:
`Tcl_AppendResult()` is deprecated. Use` Tcl_SetResult()` if you can. You could do a one-liner, something like:

`Tcl_SetResult(interp, find_timer(timer, argv[1]) ? "1" : "0", TCL_STATIC);`